File: Assignment 4/Q3.c
/* Write a Program which Accept Number from User and Display all its non factors
   ex.
   Input : 12
   Output: 5    7   8   9   10  11

   Input : 10
   Output: 3    4   6   7   8   9
*/

#include<stdio.h>

void NonFact(int iNo)
{
    int i = 0;

    for(i=2; i<iNo; i++)
    {   
        if(iNo % i != 0)
        {
            printf("%d\t",i);
        } 
    }   
}

int main()
{
    int iValue = 0;

    printf("Enter Number :");
    scanf("%d",&iValue);

    NonFact(iValue);

    return 0;
}

File: Assignment 21/Q3.java
// Write a Program to Display all Factors of a given Number.

class Logic
{
    void displayFactors(int num)
    {
        int i = 0;

        for(i = 1; i <= num ; i++)
        {
            if((num % i) == 0)
            {
                System.out.println(i);
            }
        }
    }
}

class Q3
{
    public static void main(String args[])
    {
        Logic obj = new Logic();
        obj.displayFactors(12);
    }
}